### ISSUE TITLE
[MHV MR / Accessibility] Focus indicator partially obscured on print/download menu items

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#122275]
### 📝 User story
As a keyboard user using the print/download actions menu button, I want the menu item with keyboard focus to be readily distinguishable from other menu items so that I can select the right item.

### 📝 Problem statement
Focused menu items lack the bottom border of the focus indicator.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Medical Records > Allergies and reactions](https://staging.va.gov/my-health/medical-records/allergies)

<img width="663" height="548" alt="Print/download menu expanded with focus on Print this list item" src="https://github.com/user-attachments/assets/e630e68d-e3bc-473f-8558-bc606476845e" />

---

#### 👣 Steps to recreate
1. Navigate to the Medical Records > Allergies and reactions page.
2. Use the Tab key to navigate to the Print or download actions menu component; select Enter to expand the menu.
3. Use the Tab key to move focus to Print this list item.
4. Notice that the bottom border of the focus indicator (yellow border) is not visible.

</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [SC 2.4.7 Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
- Adjust the implementation of the print/download actions menu button to show all borders of the focus indicator.
- This should be a simple Z-index adjustment on the active/focused element.

### 🔎 Supporting resources
- [Focus management](https://design.va.gov/accessibility/focus-management)

### ✅ Acceptance criteria
- [ ] The system displays the focus indicator around all edges of each item in the print/download actions menu button.
- [ ] The user can navigate through items in the print/download actions menu button and see the full focus indicator around each menu item.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

### 🔎 General resources
- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by steveg-IAT** [comment]
Related/Duplicate issue to Medications product ticket:  https://github.com/department-of-veterans-affairs/va.gov-team/issues/123468
